tutorials/intro/plot_introduction.py

```diff
@@ -1,404 +1,293 @@
+# | 4        | visual stimulus (checkerboard) to the right visual field |
+# +----------+----------------------------------------------------------+
+# | 5        | smiley face (catch trial)                                |
+# +----------+----------------------------------------------------------+
+# | 32       | subject button press                                     |
+# +----------+----------------------------------------------------------+
+
+event_dict = {'auditory/left': 1, 'auditory/right': 2, 'visual/left': 3,
+              'visual/right': 4, 'smiley': 5, 'buttonpress': 32}
+
+###############################################################################
+# Event dictionaries like this one are used when extracting epochs from
+# continuous data; the ``/`` character in the dictionary keys allows pooling
+# across conditions by requesting partial condition descriptors (i.e.,
+# requesting ``'auditory'`` will select all epochs with Event IDs 1 and 2;
+# requesting ``'left'`` will select all epochs with Event IDs 1 and 3). An
+# example of this is shown in the next section. There is also a convenient
+# :func:`~mne.viz.plot_events` function for visualizing the distribution of
+# events across the duration of the recording (to make sure event detection
+# worked as expected):
+
+fig = mne.viz.plot_events(events, event_id=event_dict, sfreq=raw.info['sfreq'])
+fig.subplots_adjust(right=0.7)  # make room for the legend
+
+###############################################################################
+# Epoching continuous data
+# ^^^^^^^^^^^^^^^^^^^^^^^^
 #
-# Scipy supports read and write of matlab files. You can save your single
-# trials with:
-
-from scipy import io  # noqa
-io.savemat('epochs_data.mat', dict(epochs_data=epochs_data), oned_as='row')
-
-##############################################################################
-# or if you want to keep all the information about the data you can save your
-# epochs in a fif file:
-
-epochs.save('sample-epo.fif', overwrite=True)
-
-##############################################################################
-# and read them later with:
-
-saved_epochs = mne.read_epochs('sample-epo.fif')
-
-##############################################################################
-# Compute evoked responses for auditory responses by averaging and plot it:
-
-evoked = epochs['aud_l'].average()
-print(evoked)
-evoked.plot(time_unit='s')
-
-##############################################################################
-# .. topic:: Exercise
+# The :class:`~mne.io.Raw` object and the events array are the bare minimum
+# needed to create an :class:`~mne.Epochs` object, which we create with the
+# :class:`mne.Epochs` class constructor. Here we'll also specify some data
+# quality constraints: we'll reject any epoch where peak-to-peak signal
+# amplitude is beyond reasonable limits for that channel type. This is done
+# with a *rejection dictionary*:
+
+reject_criteria = dict(mag=4e-12,     # 4000 fT
```

Comment:
4000e-15? Just to be a bit more explicit about the units

Reply:
hmm... that's why I put in the comments.  On your logic should grads be `4000e-13`? or `4000e-15 / 1e-2`?

Reply:
Grads should be 4000e-13 if you were to go by my logic.